content/dictionary/test-case.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: test case
-definition: ''
+definition: 'Document that describes the step by step process of how to test an application.'
```

Comment:
```suggestion
definition: 'Document that describes the step by step process of how to test an application.'
sources:
- sourceurl: https://softwaretestingfundamentals.com/test-case/
```

I noticed you added thesource url in the PR - can you add it here as well (I think you can click "accept suggestion" or something like that below)?